File: programmers/level1/시저암호/javascript/20221021/solution.test.js
import solution from './solution';

test('solution', () => {
  expect(solution('AB', 1)).toEqual('BC');
});

test('solution', () => {
  expect(solution('z', 1)).toEqual('a');
});

test('solution', () => {
  expect(solution('a B z', 4)).toEqual('e F d');
});

File: programmers/level1/자릿수더하기/java/20221018/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {

    @Test
    void sum() {
        Solution solution = new Solution();

        assertEquals(6, solution.sum(123));
        assertEquals(24, solution.sum(987));
    }

}
